airqo-platform/AirQo-frontend
# Enhancements for forgot & reset pages

- What does this PR do?

Adds input validations to the reset password page
Updates Redux components for forgot password page
Confirmation messages on the Forgot Password page

- How do I test it out?
cd AirQo-frontend/netmanager
npm install 

For Windows:
npm run stage-pc

For Mac:
npm run stage-mac

- Which JIRA card is associated?
None right now

- Any other considerations?
As we run in the staging environment, please ensure that you first create an account under the user section of the platform and then change that new user's password.
Do not change the one for the super admin which others might want to use when accessing.
